### ISSUE TITLE
How about a little help not just for the subscriber ?

### ISSUE THREAD

**Issue by daguitosama** [genmon/aboutfeeds#31]
Hey man i just landed the aboutfeeds from a feed (who would gess that 😂). Im building my own blog, im seeing different blogs feeds trying to decode how those feeds should be crafted, i think it would be nice to add to aboutfeeds a little bit of: 
> Hey if you are new to the feed crafting stuff you should check out this resource.

What you think ?

Any suggestions about how to do a good feed will be more them welcome.

**Comment by jasikpark** [comment]
Useful links:
 
- https://indieweb.org/RSS
- https://indieweb.org/Atom
- https://indieweb.org/feed_file

**Comment by jasikpark** [comment]
yeah i use Hugo's RSS generator - https://gohugo.io/templates/rss/

**Comment by stevenjmesser** [comment]
Yeah, we should add a bit about adding RSS to your site.
